docs/setup-mac.md

```diff
@@ -97,3 +97,26 @@ See [./setup-gmail.md](./setup-gmail.md) for instructions on setting up Gmail to
     ```sh
     yarn workspace server run db:seed
     ```
+
+## Datadog
+
+It's possible to send tracing data from your local development instance to datadog.  To do so, you will need to:
+
+1. Install a [Datadog Agent](https://docs.datadoghq.com/getting_started/agent/) locally and give it credentials to send data to the USDR datadog instance.
+1. Configure the datadog tracing library with environment variables
+
+### Install the Datadog Agent
+
+Datadog provides agent installation instructions within the [datadog.hq app](https://app.datadoghq.com/account/settings/agent/latest?_gl=1%2Ancclch%2A_gcl_aw%2AR0NMLjE2OTI3NDkyNzEuQ2owS0NRand1WkduQmhEMUFSSXNBQ3hiQVZqQS13YTlHVFFORGxQaVlFWDQzb1hiWTdHYzNxTFpScXhQam84UFg3cnRqTHZxT1UxSWdNY2FBbzJRRUFMd193Y0I.%2A_gcl_au%2AMTUyNDczOTE5Mi4xNjkyMjQxMDkw%2A_ga%2AMTQ3MTcxNzMzMy4xNjkzMDIxMDQ2%2A_ga_KN80RDFSQK%2AMTY5MzE5MDg2Mi41LjEuMTY5MzE5MDkxNC4wLjAuMA..%2A_fplc%2ARnhXc1BjdFZjcmF6dFZmSmpJNDlXRDUzeXVpdVdtZWI4eDhnazhvYVdpUkI1dWRESFBwVzN1QnNIRXBDVHllJTJCdVdBZUxpJTJGJTJCWCUyRjNnUWVnQWgycWVSb3IyR2tyeWp6T0pSR01oc2YlMkZHWkN5MWd6RTRXS1lWdU80bnR3JTJGJTJCb2clM0QlM0Q.&platform=macos).
```

Comment:
I would very much prefer we direct people to use the Docker-based agent rather than having them install a Mac or Windows system service, as it will be much easier for us to support any issues someone might run into if we can assume a Docker-based environment.

We could even provide a Dockerfile and some simple commands to start/stop the docker-based agent when needed, which I think would be really useful to future contributors.

Reply:
I don't have a problem with us taking the stance to prefer docker workflows.  That's actually why I thought this particular documentation was a good fit for this mac-specific local development document.

I also think it makes a lot of sense to document or add configuration for the docker workflow, but that's not what I did so far.  I just thought I would offer to capture some notes from what I did work out, in case it's helpful.

If it's not, we don't have to land anything for now.

Reply:
My preference would be to generally only do a documentation write-up for recommended patterns/procedures we'd like contributors to be able to repeat with the benefit of documented steps. 

For context, newer volunteers tend to refer to the `setup-mac.md` documentation when they're getting started with the project, so even though the intent for this might be "this is something that worked for me in a specific scenario", it's likely to be interpreted by others as "steps I should follow in order to fully configure my environment for local development".